### PR TITLE
Remove dependency on nette/security

### PR DESCRIPTION
**Pull request by enumag**
As far as I can tell nette/security seems to be optional everywhere in nette/application.

### CODE REVIEW

**Comment by dg** [comment]
Can you change commit message to smthng like `composer: removed …`

**Comment by enumag** [comment]
@dg Done.

**Comment by dg** [comment]
Btw it is required by storeRequest() & restoreRequest()

**Comment by enumag** [comment]
@dg Added some conditions to make them work without user.

**Comment by dg** [comment]
Did you test it?

**Comment by enumag** [comment]
Oh right... can't use getUser because it throws an exception... Added a test for storeRequest as well.

Can't really test it in a real application though. I use my own implementation of these methods to make them work with object parameters.

**Comment by Majkl578** [comment]
Needs rebase after force-pushes... :(

**Comment by dg** [comment]
Thanks, merged

File: src/Application/UI/Presenter.php [hunk 1]
```diff
@@ -1080,7 +1080,7 @@ public function storeRequest($expiration = '+ 10 minutes')
 			$key = Nette\Utils\Random::generate(5);
 		} while (isset($session[$key]));
 
-		$session[$key] = [$this->getUser()->getId(), $this->request];
+		$session[$key] = [$this->user ? $this->user->getId() : null, $this->request];
```
Comment (lookyman) [inline src/Application/UI/Presenter.php]: `NULL`

Comment (enumag) [inline src/Application/UI/Presenter.php]: Yeah, @dg already fixed that when he merged it. :-)

**Pull request merged**